config/newrelic.ini

```diff
@@ -23,4 +23,4 @@
 # Here are the settings that are common to all environments.
 
 [newrelic]
-error_collector.expected_classes = werkzeug.exceptions:Forbidden
\ No newline at end of file
+error_collector.expected_classes = werkzeug.exceptions:Forbidden werkzeug.exceptions:MethodNotAllowed
```

Comment:
Weren't you saying that you didn't want to ignore valid errors like if a user is trying to update a resource and calls the wrong method?

Reply:
This is the better of 2 bad options; either ignore this and get notified of all other error types (where users are trying to do things they are not permitted to do), or miss valid important alerts because there are too many not actionable alerts...